server/services/crmDisplayService.test.ts

```diff
@@ -203,73 +125,76 @@ describe('CRM Display Service', () => {
         formDetails: {
           ...crm5Response.formDetails,
           hasPreviousApplication: '',
-          previousApplicationRef: '',
           appealedPrevDecision: '',
-          appealedPrevDecisionDetails: '',
           urgent: '',
           urgencyReason: '',
+          Firm: {
+            firmAddress: '1 SOME PLACE',
+            firmName: '',
+            firmPhone: '',
+            firmSupplierNo: '',
+            firmContactName: '',
+            firmSolicitorName: '',
+            firmSolicitorRef: '',
+          },
         },
         evidenceFiles: {
           files: [],
         },
-        FurtherInformation: {
-          files: [],
-        },
+        FurtherInformation: [],
       }
 
-      const result = crmDisplayService.getSections('crm5', 'general-information', customResponse)
+      const result = crmDisplayService.getSections('crm5', 'firm-details', customResponse)
 
       expect(result).toEqual([
         {
-          sectionId: 'general-information',
-          subsections: [{ fields: [], title: 'General Information' }],
-          title: 'General Information',
+          sectionId: 'firm-details',
+          subsections: [
+            {
+              title: 'Firm Details',
+              fields: [],
+            },
+            {
+              title: "Solicitor's Details",
+              fields: [
+                {
+                  apiField: 'Firm.firmAddress',
+                  label: 'Address/DX',
+                  type: 'multiline',
+                  value: '1 SOME PLACE',
+                },
+              ],
+            },
+            {
+              customDisplay: {
+                apiField: 'CaseDetails',
+                name: 'crm5CaseDetails',
+                value: '',
+              },
+              fields: undefined,
+            },
+          ],
+          title: 'Firm Details',
         },
       ])
     })
 
-    it('should handle crm section with no data in crm response', () => {
+    it('should skip sections with no data in crm response', () => {
       const result = crmDisplayService.getSections('crm5', 'court-of-appeal-funding', crm5Response)
 
-      expect(result).toEqual([
-        {
-          sectionId: 'court-of-appeal-funding',
-          subsections: [{ fields: [], title: 'Court of Appeal Funding' }],
-          title: 'Court of Appeal Funding',
-        },
-      ])
-    })
-
-    it('should return crm section for conditions met', () => {
```

Comment:
This test case needs to be reinstated - 'should return crm section for conditions met'